php/Ch01-CRUD/taste-of-riak.php

```diff
@@ -1,63 +1,92 @@
-<?php 
-require_once 'riak-php-client/src/Basho/Riak/Riak.php';
-require_once 'riak-php-client/src/Basho/Riak/Bucket.php';
-require_once 'riak-php-client/src/Basho/Riak/Exception.php';
-require_once 'riak-php-client/src/Basho/Riak/Link.php';
-require_once 'riak-php-client/src/Basho/Riak/MapReduce.php';
-require_once 'riak-php-client/src/Basho/Riak/Object.php';
-require_once 'riak-php-client/src/Basho/Riak/StringIO.php';
-require_once 'riak-php-client/src/Basho/Riak/Utils.php';
-require_once 'riak-php-client/src/Basho/Riak/Link/Phase.php';
-require_once 'riak-php-client/src/Basho/Riak/MapReduce/Phase.php';
-
-$client = new Basho\Riak\Riak('127.0.0.1', 8098);
+<?php
+
+include_once 'vendor/autoload.php';
+
+use Basho\Riak;
+use Basho\Riak\Node;
+use Basho\Riak\Command;
+
+$node = (new Node\Builder)
+        ->atHost('127.0.0.1')
+        ->onPort(8098)
+        ->build();
+
+$riak = new Riak([$node]);
 
 // Note: Use this line instead of the former if using a local devrel cluster
-//$client = new Basho\Riak\Riak('127.0.0.1', 10018);
+// $node = (new Node\Builder)->atHost('127.0.0.1')->onPort(10018)->build();
 
 // Creating Objects In Riak
-print('Creating Objects In Riak...\n');
-
-$myBucket = $client->bucket('test');
+('Creating Objects In Riak...' . PHP_EOL);
 
 $val1 = 1;
-$obj1 = $myBucket->newObject('one', $val1);
-$obj1->store();
-
 $val2 = 'two';
-$obj2 = $myBucket->newObject('two', $val2);
-$obj2->store();
+$val3 = ['myValue' => 3];
+
+$bucket = new Riak\Bucket('testBucket');
+$location1 = new Riak\Location('one', $bucket);
+$location2 = new Riak\Location('two', $bucket);
+$location3 = new Riak\Location('three', $bucket);
+
+
+$storeCommand1 = (new Command\Builder\StoreObject($riak))
+                    ->buildObject($val1)
+                    ->atLocation($location1)
+                    ->build();
+$storeCommand1->execute();
+
 
-$val3 = array('myValue' => 3);
-$obj3 = $myBucket->newObject('three', $val3);
-$obj3->store();
+$storeCommand2 = (new Command\Builder\StoreObject($riak))
```

Comment:
Similar to what I said here: https://github.com/basho/basho_docs/commit/50dd8a74f25dfa9fd301ad682dc2dc0b5441c2ae#commitcomment-10875051

You can reuse the builder when you are doing the same command repeatedly.